Package.swift

```diff
@@ -18,83 +18,46 @@ import PackageDescription
 import Foundation
 
 var packageDependencies: [Package.Dependency] = [
-  // Official SwiftProtobuf library, for [de]serializing data to send on the wire.
-  .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMajor(from: "1.5.0")),
-  
-  // Command line argument parser for our auxiliary command line tools.
-  .package(url: "https://github.com/kylef/Commander.git", .upToNextMinor(from: "0.8.0")),
-  
-  // SwiftGRPCNIO dependencies:
-  // Transitive dependencies
-  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", .upToNextMajor(from: "1.0.0")),
-  .package(url: "https://github.com/apple/swift-nio-nghttp2-support.git", .upToNextMajor(from: "1.0.0")),
-  // Main SwiftNIO package
-  .package(url: "https://github.com/apple/swift-nio.git", .upToNextMajor(from: "1.12.0")),
-  // HTTP2 via SwiftNIO
-  .package(url: "https://github.com/apple/swift-nio-http2.git", .upToNextMinor(from: "0.2.1"))
-]
-
-var cGRPCDependencies: [Target.Dependency] = []
+// Official SwiftProtobuf library, for [de]serializing data to send on the wire.
+.package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMajor(from: "1.5.0")),
 
-#if os(Linux)
-let isLinux = true
-#else
-let isLinux = false
-#endif
-
-// On Linux, Foundation links with openssl, so we'll need to use that instead of BoringSSL.
-// See https://github.com/apple/swift-nio-ssl/issues/16#issuecomment-392705505 for details.
-// swift build doesn't pass -Xswiftc flags to dependencies, so here using an environment variable
-// is easiest.
-if isLinux || ProcessInfo.processInfo.environment.keys.contains("GRPC_USE_OPENSSL") {
-  packageDependencies.append(.package(url: "https://github.com/apple/swift-nio-ssl-support.git", from: "1.0.0"))
-} else {
-  cGRPCDependencies.append("BoringSSL")
-}
```

Comment:
@nakkht I think is the source of your build failure: we're now _always_ depending on BoringSSL instead of conditionally depending on the platform.

Reply:
@glbrntt does the same apply when using `Package@swift-4.2`?

Reply:
I believe so, yes.

Reply:
@glbrntt how about `ProcessInfo.processInfo.environment.keys.contains("GRPC_USE_OPENSSL")`? Is it still have to be included?

Reply:
I believe so, yes.